src/components/Renderers/createChart.tsx

```diff
@@ -83,9 +83,10 @@ const createChart = (
       ? chart.tooltip.labelName ?? snakeToSentence(chart.props.y as string)
       : '';
 
-  return (
+  return chart.props?.dataComponent && functions.dataComponentMapper ? (
```

Comment:
We can put this into props (warpper does it nicely) so we have less duplicate code :)